src/legacy/core_plugins/kbn_vislib_vis_types/public/controls/point_series/series.js

```diff
@@ -1,102 +0,0 @@
-/*
- * Licensed to Elasticsearch B.V. under one or more contributor
- * license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright
- * ownership. Elasticsearch B.V. licenses this file to you under
- * the Apache License, Version 2.0 (the "License"); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-import _ from 'lodash';
-import { uiModules } from 'ui/modules';
-import vislibSeriesTemplate from './series.html';
-import { safeMakeLabel } from 'ui/agg_types/agg_utils';
-
-const module = uiModules.get('kibana');
-
-module.directive('vislibSeries', function () {
-  return {
-    restrict: 'E',
-    template: vislibSeriesTemplate,
-    replace: true,
-    link: function ($scope) {
-      function makeSerie(id, label) {
-        const last = $scope.series[$scope.series.length - 1];
-        return {
-          show: true,
-          mode: last ? last.mode : 'normal',
-          type: last ? last.type : 'line',
-          drawLinesBetweenPoints: last ? last.drawLinesBetweenPoints : true,
-          showCircles: last ? last.showCircles : true,
-          interpolate: last ? last.interpolate : 'linear',
-          lineWidth: last ? last.lineWidth : 2,
-          data: {
-            id: id,
-            label: label
-          },
-          valueAxis: last ? last.valueAxis : $scope.editorState.params.valueAxes[0].id
-        };
-      }
-
-      $scope.series = $scope.editorState.params.seriesParams;
-      $scope.$watch(() => {
-        return $scope.editorState.aggs.map(agg => {
-          return safeMakeLabel(agg);
-        }).join();
-      }, () => {
-        const schemaTitle = $scope.vis.type.schemas.metrics[0].title;
```

Comment:
**note:** moved to https://github.com/elastic/kibana/pull/43772/files?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx&hide-deleted-files=true#diff-7b90f2dbd1b54e2bc7d07fd3497e08d5R192